.github/workflows/CI.yml

```diff
@@ -73,17 +73,7 @@ jobs:
           DJANGO_SETTINGS_MODULE: sampl.settings_dev
 
       - name: Upload test coverage
-        uses: codecov/codecov-action@v2.1.0
+        uses: codecov/codecov-action@v2.0.3
         with:
           fail_ci_if_error: true
           directory: app/
-
-      - name: Run linter
```

Comment:
so if someone forgets to install pre-commit?

Reply:
We will have pre-commit running on the CI so developers do need to have it installed
https://results.pre-commit.ci/repo/github/327929427